README.rst

```diff
@@ -70,9 +70,74 @@ We appreciate any help!
 BUILD & DEVELOPMENT
 -------------------
 
-Run ``python3 setup.py install``
+1. Clone the *coala-html* repository
 
-Note: You need to have bower installed before you may install the python packages.
+::
+
+  $ git clone https://github.com/coala/coala-html.git
+
+2. Install latest version of **Node.js**
+
+- *Configure Node.js PPA*
+
+::
+
+    $ sudo apt-get install curl python-software-properties
+    $ curl -sL https://deb.nodesource.com/setup_11.x | sudo bash -
+
+- *Install Node.js and npm*
+
+::
+
+  $ sudo apt-get install nodejs
+
+- *Check Node.js and npm version*
+
+  You can find more details about current version on node.js `official website
+  <https://nodejs.org/>`__.
+
+::
+
+    $ node -v
+    v11.6.0
+
+    $ npm -v
+    6.5.0
+
+3. Install **bower** to build static web pages
+
+::
+
+    $ sudo npm install -g bower
+
+4. Change the directory to *coala-html* and run following command to install dependencies
+
+::
+
+    $ python setup.py install
+
+5. Install **coala-bears**
+
+::
+
+    $ pip install coala-bears
+
+6. Allow permissions to **root user** to install node packages
+
+::
+
+    $ sudo chown -R $USER:$GROUP ~/.npm
+    $ sudo chown -R $USER:$GROUP ~/.config
+    $ sudo chown -R $USER:$GROUP ~/.cache/bower/registry/registry.bower.io/lookup/
+    $ sudo chown -R $USER:$GROUP ~/.cache/bower/packages
+
+7. Run **coala-html** to display results from *coala-json*
+
+::
+
+$ coala-html --dir ..\Tests
```

Comment:
Talking about windows, shouldn't instructions be added for windows as well?

Reply:
> Backslash? Above commands were for Linux(Ubuntu), looking at the backslash this seems to be a Windows command. Again, should be consistent with rest of the installation flow.

This command too runs on linux

Reply:
> Talking about windows, shouldn't instructions be added for windows as well?

Instructions for windows can be added, but it will take more space of README.rst. I was thinking to add an extra file to repository name `Installation Instructions.md` where we can provide a detailed instructions for ubuntu and windows.
and then provide that file link in this `README.rst` file

Reply:
> This command too runs on linux

Nevertheless, this looks out of place and inconsistent.
> I was thinking to add an extra file

Do we really need a separate file? Why don't you try it here first? I don't think it would take an unacceptable amount of space. Also including the instructions for just one OS won't be right (unless its done somewhere else too). Although, a maintainer would be able to better confirm this.

Reply:
Ok, I will put instructions for windows as well as ubuntu in same file by making two sections each having its own set of instructions and will also provide the link of installation instructions for other ubuntu distributions like redHat, arch etc.

Reply:
Yeah, you can split them into multiple files later, if required or suggested